### DISCUSSION TITLE
Min ignored when using numeric

### DISCUSSION THREAD

**Discussion by jynxy** [livewire/livewire#7994] [7. Bugs (archived)]
### Livewire version

v3.4.6

### Laravel version

10.45.1

### Which PHP version are you using?

PHP 8.1

### Steps To Reproduce

When using min and numeric, the min is ignored when entering numbers, if you type letters works as expected.

Use basic demo on wirebox. just added validation and error message.

https://wirebox.app/b/4mpn5

Try type in abc, this will show both its no numeric and not a min of 6.

Now type in 123, this will allow you to complete this.

Then try entering just 1, this will show its not a min of 6.

### Repository or Wirebox URL

https://wirebox.app/b/4mpn5

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by joshhanley** [comment]
@jynxy please don't open a second discussion for the same problem. Closing it so we keep working on the original discussion #7977.

Please paste your wirebox in that discussion as asked and also it is private so make it public so we can access it.
